src/utils/legacy-settings.js

```diff
@@ -130,12 +130,21 @@ function deserializeSettingForLegacy(data, settingId) {
       const hideOfflineFollowedChannels = data[LegacySettingIds.HIDE_OFFLINE_FOLLOWED_CHANNELS] || false;
       const autoExpand = data[LegacySettingIds.AUTO_EXPAND_CHANNELS] || false;
 
-      let flags = setFlag(0, SidebarFlags.FEATURED_CHANNELS, !hideFeaturedChannels);
+      let flags = setFlag(
+        SidebarFlags.FEATURED_CHANNELS,
```

Comment:
this looks to change the behavior of the import. i think you want:

```
let flags = setFlag(0, SidebarFlags.FEATURED_CHANNELS | SidebarFlags.RECOMMENDED_CHANNELS | SidebarFlags.SIMILAR_CHANNELS, !hideFeaturedChannels);
```

the `0` is the default value of the setting, which is all off by default. this is OK because we only configure bits related to the import. new bits which get added, which are not changed in the changed bits below, will automatically be added on migration